requirements.txt

```diff
@@ -43,7 +43,7 @@ tensorboard==2.11.0
 tensorboard-data-server==0.6.1
 tensorboard-plugin-wit==1.8.1
 tensorflow==2.11.0
-tensorflow-io==0.27.0
+tensorflow-io==0.28.0
```

Comment:
Why did we increase the version requirement?

Reply:
Currently, there is a warning during `pip install -r requirements.txt` stating the incompatibility with the current tensorflow version. `tensorflow-io` needs to be 0.28.0 for the right match

From logs,
`ERROR: tensorflow-io 0.27.0 has requirement tensorflow-io-gcs-filesystem==0.27.0, but you'll have tensorflow-io-gcs-filesystem 0.28.0 which is incompatible.`